UI/src/model/selectedLaneModelling.js

```diff
@@ -4,16 +4,15 @@
     var lanesFetched = [];
     var selectedRoadlink = null;
     var assetsToBeExpired = [];
-    var assetsToBeRemoved = [];
     var self = this;
     var linksSelected = null;
     var currentLane;
 
-    var initial_road_number;
-    var initial_road_part_number;
-    var initial_distance;
-    var end_road_part_number;
-    var end_distance;
+    var roadNumber;
+    var roadPartNumber;
+    var startAddrMValue;
+    var endRoadPartNumber;
+    var endDistance;
```

Comment:
Isnt this endDistance the endAddrMValue ?

Reply:
Yes, are you suggesting to change the name?

Reply:
Ok then.